firstvoices/jwt_auth/authentication.py

```diff
@@ -15,11 +15,10 @@ def extract_bearer_token(auth):
     try:
         scheme, token = auth.split()
     except ValueError:
-        raise exceptions.AuthenticationFailed("Invalid format for authorization header")
+        raise exceptions.NotAuthenticated("Invalid format for authorization header")
     if scheme != "Bearer":
-        raise exceptions.AuthenticationFailed("Authorization header invalid")
-    if not token:
-        raise exceptions.AuthenticationFailed("No token found")
+        raise exceptions.NotAuthenticated("Authorization header invalid")
+
```

Comment:
I removed the `if not token case` here, because in that situation it would raise a ValueError on line 16, so this code was unreachable